appservice/src/main/java/nu/yona/server/subscriptions/rest/UserController.java

```diff
@@ -501,15 +503,35 @@ public PostPutUserDto(@JsonProperty("firstName") String firstName, @JsonProperty
 		}
 	}
 
+	enum Purpose
```

Comment:
I think the purpose of this enum is to get an overview of the possible states a user resource can have. So I think `PresentationState` is maybe a better name.